specs/superchain-upgrades.md

```diff
@@ -308,10 +309,35 @@ The Delta upgrade consists of a single consensus-layer feature: [Span Batches](.
 
 The Delta upgrade uses a *L2 block-timestamp* activation-rule, and is specified only in the rollup-node (`delta_time`).
 
-## Eclipse
+## Ecotone
 
-Name of the next upgrade after Delta. Placeholder for development coordination.
+The Ecotone upgrade contains the Dencun upgrade from L1, and adopts EIP-4844 blobs for data-availability.
+
+Dencun Upgrade:
+
+- Cancun (Execution Layer):
+  - [EIP-1153: Transient storage opcodes](https://eips.ethereum.org/EIPS/eip-1153)
+  - [EIP-4844: Shard Blob Transactions](https://eips.ethereum.org/EIPS/eip-4844)
+    - [Blob transactions are disabled](./exec-engine.md#ecotone-disable-blob-transactions)
+  - [EIP-4788: Beacon block root in the EVM](https://eips.ethereum.org/EIPS/eip-4788)
+    - [The L1 beacon block root is embedded into L2](./exec-engine.md#ecotone-beacon-block-root)
+    - [The Beacon roots contract deployment is automated](./derivation.md#ecotone-beacon-block-roots-contract-deployment-eip-4788)
+  - [EIP-5656: MCOPY - Memory copying instruction](https://eips.ethereum.org/EIPS/eip-5656)
+  - [EIP-6780: SELFDESTRUCT only in same transaction](https://eips.ethereum.org/EIPS/eip-6780)
+  - [EIP-7516: BLOBBASEFEE opcode](https://eips.ethereum.org/EIPS/eip-7516)
```

Comment:
This is underspecified - when you say not applicable to L2, does that mean it will always return 0?

Reply:
My intention was to leave it functional, but let it return what it would have returned if there was never any blob tx included. We should probably explain in the spec that it then actually return `1`, not `0`. When running `CalcBlobFee(excessBlobGas=0)` that's what you get in geth.

Reply:
Do you think we should specify that it will always return 1?